responses/tests/test_responses.py

```diff
@@ -1838,6 +1838,34 @@ def run():
         run()
         assert_reset()
 
+    def test_real_send_argument(self):
+        def run():
+            # the following mock will serve to catch the real send request from another mock and
+            # will "donate" `unbound_on_send` method
+            mock_to_catch_real_send = responses.RequestsMock(
+                assert_all_requests_are_fired=True
+            )
+            mock_to_catch_real_send.post(
+                re.compile(r"http://localhost:7700.*"), status=500
+            )
+
+            with responses.RequestsMock(
+                assert_all_requests_are_fired=True,
+                real_adapter_send=mock_to_catch_real_send.unbound_on_send(),
+            ) as r_mock:
+                r_mock.add_passthru(re.compile(r"http://localhost:7700.*"))
+
+                r_mock.add(responses.POST, "https://example.org", status=200)
+
+                response = requests.post("https://example.org")
+                assert response.status_code == 200
+
+                response = requests.post("http://localhost:7700/indexes/test/documents")
+                assert response.status_code == 500
```

Comment:
I see you're using the URLs I provided in my example in #670 for your unit test here. These URLs are used as an example for a `meilisearch` instance. Would it be more clear in the tests to use a generic `example.org` or something similar as a passthrough instead of this ?

This way other people won't wonder why it's using `localhost:7700` and talks about indexing documents